dashbuilder/dashbuilder-client/dashbuilder-widgets/src/main/java/org/dashbuilder/client/widgets/dataset/editor/workflow/create/PrometheusDataSetBasicAttributesWorkflow.java

```diff
@@ -0,0 +1,76 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dashbuilder.client.widgets.dataset.editor.workflow.create;
+
+import javax.enterprise.context.Dependent;
+import javax.enterprise.event.Event;
+import javax.inject.Inject;
+import javax.validation.ConstraintViolation;
+
+import com.google.gwt.editor.client.SimpleBeanEditorDriver;
+import org.dashbuilder.client.widgets.dataset.editor.attributes.DataSetDefBasicAttributesEditor;
+import org.dashbuilder.client.widgets.dataset.editor.driver.PrometheusDataSetDefAttributesDriver;
+import org.dashbuilder.client.widgets.dataset.event.CancelRequestEvent;
+import org.dashbuilder.client.widgets.dataset.event.SaveRequestEvent;
+import org.dashbuilder.client.widgets.dataset.event.TestDataSetRequestEvent;
+import org.dashbuilder.dataset.client.DataSetClientServices;
+import org.dashbuilder.dataset.client.editor.PrometheusDataSetDefAttributesEditor;
+import org.dashbuilder.dataset.def.PrometheusDataSetDef;
+import org.dashbuilder.validations.DataSetValidatorProvider;
+import org.jboss.errai.ioc.client.container.SyncBeanManager;
+
+/**
+ * <p>Prometheus Data Set Editor workflow presenter for setting data set definition basic attributes.</p>
+ */
+@Dependent
+public class PrometheusDataSetBasicAttributesWorkflow extends DataSetBasicAttributesWorkflow<PrometheusDataSetDef, PrometheusDataSetDefAttributesEditor> {
+
+    @Inject
+    public PrometheusDataSetBasicAttributesWorkflow(final DataSetClientServices clientServices,
+                                                    final DataSetValidatorProvider validatorProvider,
+                                                    final SyncBeanManager beanManager,
+                                                    final DataSetDefBasicAttributesEditor basicAttributesEditor,
+                                                    final Event<SaveRequestEvent> saveRequestEvent,
+                                                    final Event<TestDataSetRequestEvent> testDataSetEvent,
+                                                    final Event<CancelRequestEvent> cancelRequestEvent,
+                                                    final View view) {
+
+        super(clientServices,
+              validatorProvider,
+              beanManager,
+              basicAttributesEditor,
+              saveRequestEvent,
+              testDataSetEvent,
+              cancelRequestEvent,
+              view);
+    }
+
+    @Override
+    protected Class<? extends SimpleBeanEditorDriver<PrometheusDataSetDef, PrometheusDataSetDefAttributesEditor>> getDriverClass() {
+        return PrometheusDataSetDefAttributesDriver.class;
+    }
+
+    @Override
+    protected Class<? extends PrometheusDataSetDefAttributesEditor> getEditorClass() {
+        return org.dashbuilder.client.widgets.dataset.editor.prometheus.PrometheusDataSetDefAttributesEditor.class;
```

Comment:
Can you import `org.dashbuilder.client.widgets.dataset.editor.prometheus.PrometheusDataSetDefAttributesEditor` instead of using its fully qualified name?